plugins/grafana-custom-plugins/grafana-dependency-plugin/src/DependencyPanel.tsx

```diff
@@ -26,12 +18,26 @@ export const DependencyPanel: React.FC<Props> = ({ options, data, width, height
   const destinationNodeNames = frame.fields.find((field) => field.name === 'destinationNodeName');
   const destinationServicePortNames = frame.fields.find((field) => field.name === 'destinationServicePortName');
   const octetDeltaCounts = frame.fields.find((field) => field.name === 'octetDeltaCount');
+  const sourceTransportPorts = frame.fields.find((field) => field.name === 'sourceTransportPort');
+  const destinationTransportPorts = frame.fields.find((field) => field.name === 'destinationTransportPort');
+  const sourceIPs = frame.fields.find((field) => field.name === 'sourceIP');
+  const destinationIPs = frame.fields.find((field) => field.name === 'destinationIP');
+  const httpValsList = frame.fields.find((field) => field.name === 'httpVals');
   
   let nodeToPodMap = new Map<string, String[]>();
   let srcToDestMap = new Map<string, Map<string, number>>();
 
   let graphString = 'graph LR;\n';
+  let styleString = '';
   let boxColor;
+  let layerLevel;
```

Comment:
Why we need an extra var `layerLevel`? Can we just reuse `options.layerFour` this boolean var?